frontend/src/exams/capture-ind-ita-base.vue

```diff
@@ -0,0 +1,304 @@
+<template>
```

Comment:
This isn't supposed to be ITA exam specific, correct? The plan is to use this same form component for all exam types, yes?

Reply:
yeah, i should rename this.